app/Http/Controllers/SubmissionsController.php

```diff
@@ -2,32 +2,31 @@
 
 namespace App\Http\Controllers;
 
+use App\Conference;
+use App\Submission;
 use Illuminate\Http\Request;
-use App\Commands\CreateSubmission;
-use App\Commands\DestroySubmission;
 
 class SubmissionsController extends Controller
 {
     public function store(Request $request)
     {
-        $talk = auth()->user()->talks()->findOrFail($request->input('talkId'));
-
-        $this->dispatch(new CreateSubmission(
-            $request->input('conferenceId'),
-            $talk->id
-        ));
-
-        return response()->json(['status' => 'success', 'message' => 'Talk Submitted']);
+        $talk = auth()->user()->talks()->findOrFail($request->input('talkId'))->current();
```

Comment:
Ahh--so since this is `current()`, which I totally didn't notice, can we name this variable `$talkRevision` so it's clearer?